theorems/T000171.md

```diff
@@ -0,0 +1,14 @@
+---
+uid: T000171
+if:
+  and:
+  - P000123: true
+  - P000137: false
+then:
+  P000039: false
```

Comment:
This is interesting. I normally prefer to reduce negations as much as possible, e.g. "locally Euclidean + Hyperconnected => Empty". But surely the actual use here is "locally Euclidean => not Hyperconnected" or its contrapositive, with the usual assumption that spaces are non-empty. So this is probably best as-is.